recipes/libvips/meta.yaml

```diff
@@ -74,27 +77,13 @@ requirements:
     - poppler
     - zlib
   run:
-    - bzip2 
-    - cairo 
-    - cfitsio 
-    - expat
-    - fftw
-    - fontconfig
-    - freetype
-    - gdk-pixbuf
-    - giflib
-    - glib
-    - hdf5
     - imagemagick
-    - jpeg
+    - gdk-pixbuf
     - libmatio
```

Comment:
nice, the scary part is that these run_exports aren't always well defined. but nice cleanup.